packages/flutter_sample/android/settings.gradle

```diff
@@ -0,0 +1,11 @@
+include ':app'
```

Comment:
double check: is `flutter_sample` for our sdk sample project?

Reply:
yes